driver/clirr-ignored-differences.xml

```diff
@@ -412,19 +412,13 @@
     <difference>
         <className>org/neo4j/driver/Driver</className>
         <differenceType>7012</differenceType>
-        <method>org.neo4j.driver.QueryTask queryTask(java.lang.String)</method>
```

Comment:
This name has previously been agreed with @AndyHeap-NeoTech. I will need to catch-up with him.

Reply:
maybe, but that should be reflected in the ADR.
Cannot we get a more aligned name though? It does not change the overall approach chosen by the Java driver.